<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonQuery">
        <property name="font">
         <font>
          <family>华文楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButtonAdd">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>增加书籍</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDelete">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>删除书籍</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonReturn">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>归还书籍</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDataset">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>登记号数据集</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
